exercises/sum.cpp

```diff
@@ -6,3 +6,19 @@
   Insert the second number: 2
   Sum: 3
 */
+#include <iostream>
+
+using namespace std;
+
+int main(){
+
+  int x , y;
+  cout << "Insert the first number" << endl;
+  cin >> x;
+  cout << "Insert the second number" << endl;
+  cin >> y;
+
+  cout <<"The sum of: " << x << " + " << y << " = " << x+y << endl; 
+
+  return 0;
+}
```

Comment:
il file `sum.exe` non va MAI caricato su github, cancellalo ed inseriscilo opportunamente in un file `.gitignore`